readme.md

```diff
@@ -74,13 +74,13 @@ Your `package.json` will then look like this:
 
 ```json
 {
-  "name": "awesome-package",
-  "scripts": {
-    "test": "ava"
-  },
-  "devDependencies": {
-    "ava": "^0.15.0"
-  }
+	"name": "awesome-package",
+	"scripts": {
+		"test": "ava"
+	},
+	"devDependencies": {
+		"ava": "^0.15.0"
+	}
```

Comment:
Could you revert these to their original two space indentation? It's how `package.json` files are formatted.